README.md

```
@@ -147,6 +135,10 @@ docker-compose up -d
 
 Navigate to **http://localhost:3000** and create your first account!
 
+> **Note:** Using the all-in-one `turboot/nimbus` image means no CORS or API URL configuration needed!
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)